config.md

```diff
@@ -49,6 +49,14 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 }
 ```
 
+### Example (Windows)
+
+```json
+"root": {
+    "path": "\\\\?\\Volume{ec84d99e-3f02-11e7-ac6c-00155d7682cf}"
```

Comment:
The [docs you link][1] give the pattern as `"\\?\Volume{GUID}\"` and say:

> All volume and mounted folder functions that take a volume GUID path as an input parameter require the trailing backslash.

before talking about a `CreateFile` caveat.  I'm not sure if the Windows runtime will always use `CreateFile` to back this property, or if `CreateFile` distinguishes between backslash-terminated or not volume GUID paths, but I expect you either want to add a trailing backslash here or add some words up above about why you don't need it.

[1]: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365248%28v=vs.85%29.aspx